docs/website/contents/explanation/consensus-protocol.md

```diff
@@ -176,4 +191,79 @@ Although headers are already validated by the `ChainSync` client, they are re-va
 This step is expected to succeed, given that the header was previously validated by the `ChainSync` client.
 
+## Chain Ordering in Ouroboros
+
+The core rule in Ouroboros protocols (including Praos) is to prefer longer chains over shorter ones. This assumes that the honest majority of stake will produce denser chains.
+
+- A chain that extends the current one is always preferable to it.
+- If two chains are equally preferable, the node sticks with its current chain.
+
+In Praos and Shelley-based eras, if chains have the same length, tiebreaking [rules](https://github.com/intersectmbo/ouroboros-consensus/blob/010b374c54f4d2f485ab114f702db6ec3b7a8f95/ouroboros-consensus-protocol/src/ouroboros-consensus-protocol/Ouroboros/Consensus/Protocol/Praos/Common.hs#L108) are applied in this order:
+
+- 1. **Operational Certificate Issue Number**:
+If both chain tips are from the same issuer and slot, the one with the higher `opcert` issue number is preferred.
+This allows a stake pool operator to replace a compromised hot key with a new one and still have their blocks take precedence.
+
+- 2. **VRF Tiebreaker**:
+If the `opcert` check is inconclusive (which is common when two pools are elected in the same slot), the chain with the lower VRF value at its tip is preferred.
+This avoids always picking the first block to arrive, which could encourage geographic centralization to reduce latency (see ["The Frankfurt Problem"](https://github.com/IntersectMBO/ouroboros-consensus/blob/40d77fdd74a9b2b2a1d112a0b836b5cb8026c88c/ouroboros-consensus-protocol/src/ouroboros-consensus-protocol/Ouroboros/Consensus/Protocol/Praos/Common.hs#L227)). The VRF value used for tiebreaking (non-range extended) is uncorrelated to the leader VRF value and typically results in a uniformly random decision. Depending on the `ChainOrderConfig` for Praos, there are two [flavors](https://github.com/intersectmbo/ouroboros-consensus/blob/010b374c54f4d2f485ab114f702db6ec3b7a8f95/ouroboros-consensus-protocol/src/ouroboros-consensus-protocol/Ouroboros/Consensus/Protocol/Praos/Common.hs#L75) that determine when this tie-breaker comparison takes place.
+    - `UnrestrictedVRFTiebreaker`: With this flavor, VRF tiebreakers are always compared. This has been the standard behavior for all eras before Conway.
+    - `RestrictedVRFTiebreaker`: This flavor restricts the VRF tiebreaker comparison to situations where the slot numbers of the chain tips differ by at most a specified maximum distance (`maxDist`).
+The primary motivation for this restriction is to favor blocks that were diffused earlier over those diffused later, even if the later block has a "better" VRF tiebreaker value. This aims to mitigate [issues](https://github.com/IntersectMBO/ouroboros-network/issues/2913) caused by poorly configured or resource-constrained pools that might diffuse blocks later.
+
+### On the Transitivity of Praos Chain Order
+
+Praos chain ordering is **not transitive**, regardless of the VRF tiebreaker flavor.
```

Comment:
This is technically true because of https://github.com/IntersectMBO/ouroboros-consensus/issues/1075#issuecomment-3035911537, but this is just a bug that should be fixed.

I think it is important to mention two aspects of the chain order here:

 - The `Ord` instance on `PraosChainSelectView` must be a total order (and in particular transitive). The fact that is currently isn't in a weird edge case (#1075) is just a bug.
 - `preferCandidate` is *not* transitive when using `RestrictedVRFTiebreaker`, and this is completely fine.